modules/ROOT/pages/osv2-tutorial.adoc

```diff
@@ -409,66 +199,61 @@ http://www.mulesoft.org/schema/mule/os http://www.mulesoft.org/schema/mule/os/cu
 ----
 
 
-== Create a Mule 3 App with an Object Store in Anypoint Studio 6
+[[create-app-studio6]]
+== Create a Mule 3.9 App with an Object Store in Anypoint Studio 6.x
 
-You can use Anypoint Studio 6 to create a Mule 3 app similar to the app you created in Design Center.
-First, you use Studio to configure an app that includes Object Store Connector. 
-Then, you deploy the app CloudHub and use a `curl` command to send a key and value, 
+You can use Anypoint Studio 6.x to create a Mule 3.9 app similar to the app you created in Studio 7.x.
+First, use Studio 6.x to configure an app that includes Object Store Connector for Mule 3. 
```

Comment:
Use bullets?

Reply:
This is the summary of/intro to the procedure.